go/libraries/doltcore/merge/merge_schema.go

```diff
@@ -118,6 +118,12 @@ func SchemaMerge(ourSch, theirSch, ancSch schema.Schema, tblName string) (sch sc
 	if err != nil {
 		return nil, sc, err
 	}
+
+	err = sch.SetPkOrdinals(ourSch.GetPkOrdinals())
```

Comment:
what if the three schemas have different pk ordinals? how to tell which one to keep?

Reply:
I believe we do not support merge between tables with competing ordinals.